helm-charts/ingress-controller/values.yaml

```diff
@@ -1,6 +1,6 @@
 service:
   # nodePort for easegress service
-  nodePort: 80
+  nodePort: 30080
```

Comment:
I would keep the Ingress port as 80 as it is the value for Kubernetes Ingress and [HTTP traffic in general](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers).

Reply:
this is nodePort... the real port out k8s to access is defined by `Service`. nodePort is fine to be any number...